src/containers/NFT/NFTHeader/styles.scss

```diff
@@ -0,0 +1,201 @@
+@import '../../shared/css/variables';
+@import '../../shared/css/table';
+
+.section {
```

Comment:
This naming feels like a global style.  I want to make sure this either gets moved into a shared location or is renamed to be more specific.

Reply:
this is being used by the token currency page as well. im not sure if we want to put it global when this style is specific only to these 2 places

Reply:
It will be made global anyway so we will have two definitions of `.section` last one will win.

Reply:
this doesn't sound like it was resolved. we should rename this, right?

Reply:
@ckniffen so should this be renamed?

Reply:
We created an issue to address this later.